src/pom.xml

```diff
@@ -1,20 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
         <version>2.2.2.RELEASE</version>
         <relativePath/>
     </parent>
-	
-   <groupId>ca.bc.gov.open</groupId>
-	<artifactId>dps</artifactId>
-	<version>0.0.1-SNAPSHOT</version>
-	<name>dps</name>
-	<packaging>pom</packaging>
-	<url></url>
+
+    <groupId>ca.bc.gov.open</groupId>
+    <artifactId>dps</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>dps</name>
+    <packaging>pom</packaging>
+    <url></url>
```

Comment:
empty? does this need a TODO?

Reply:
@akroon3r , yes I think need a TODO for the URL ,I missed in my initial commit.

Reply:
that was me reformatting the file

Reply:
that should be url to the git repo actually